### PR TITLE
Removed redundant code, and increased fontsize

### PR DESCRIPTION
**Pull request by correaswebert**
Cleaned up some code. Also styled up the pages a little, making it a little more legible.

### CODE REVIEW

**Comment by correaswebert** [comment]
Screenshot_20210212_220633
Screenshot_20210212_220628
Screenshot_20210212_220617
Screenshot_20210212_220540
Screenshot_20210212_220528

File: package.json [hunk 1]
```diff
@@ -1,22 +1,21 @@
 {
   "name": "website-welcome",
   "version": "1.0.0",
-  "description": "https://welcome.realdevsquad.com/",
-  "main": "index.js",
+  "description": "Welcome website of RealDevSquad",
```
Comment (swarajpure) [inline package.json:4]: Where is this file coming from? 🤔 
This is a simple HTML, CSS, JS repo. Node is not required here.
Please delete this one

**Review by swarajpure — CHANGES_REQUESTED** [review]
Good job in there! A lot of details covered! 🎉 

Also, I guess you've worked on develop branch. For this repo, the default repo was main 😅

I know this confusion shouldn't have been there. I'll take a look at the branches thing tomorrow. 

Till then please have a look at my comments

Thank you! 😁

File: package.json (hunk 1 above)
Comment (correaswebert) [reply package.json:4]: Yeah even I'm not sure of this one... It was already present there, so just removed the entry point.

File: css/pages/coc.css [hunk 2]
```diff
@@ -1,34 +1,33 @@
...
+
 .coc-header {
-  width: 90%;
   text-align: left;
   font-weight: 550;
-  padding: 20px;
- }
-
+  padding: 1.5em;
+  text-align: justify;
```
[the remark this answers is not in the document]
Comment (correaswebert) [reply css/pages/coc.css:18]: You mean the ones in `main article` and `.coc-header` ?

**Review by swarajpure — APPROVED** [review]
Great! Was very much needed! Thank you 🎉

**Review by Rucha1499 — APPROVED** [review]
Thank you! 🎉

**Pull request merged**